assignments/array-methods.js

```diff
@@ -56,28 +56,74 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+function totalName (x) {fullName.push(`${x.first_name}` + ' ' + `${x.last_name}`);}
+runners.forEach(totalName)
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+function totalName (x) {allCaps.push(`${x.first_name.toUpperCase()}`)}
+runners.map(totalName);
 console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
 let largeShirts = [];
+function LS (x) {if (x.shirt_size === 'L') { largeShirts.push(x);
```

Comment:
Your JS spacing is a bit off. 

When you get a chance review this: https://www.w3schools.com/js/js_conventions.asp